supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -39,6 +39,25 @@ Some tasks in this procedure require root privileges, which you can get temporar
 .Additional resources
 * link:https://www.redhat.com/sysadmin/difference-between-sudo-su[Exploring the differences between sudo and su commands in Linux]
 
+[[ellipses-in-yaml-code-blocks]]
+== Ellipses in YAML code blocks
+
+Use the pound sign (`#`) to comment out an ellipsis in YAML code blocks.
```

Comment:
```suggestion
Use the number sign (`#`) to comment out an ellipsis in YAML code blocks.
```

See the "Punctuation and symbols" section of _IBM Style_. The pound sign is a different symbol.

Reply:
Strange. I don't see the change yet, although I pushed the commit.